components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -244,26 +246,70 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
     }, []);
 
     const accData: AccordionItemType[] = Object.entries(data).map(([accordionKey, accordionData]) => {
-        const items = accordionData.items.map((item) => (
-            <AccordionItem
-                key={`${accordionKey}-item_${item.id}`}
-                iconColor={getItemColor(item.status)}
-            >
-                <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
-                <p>{item.description}</p>
-            </AccordionItem>
-        ));
+        const chipsInfo: { [key in ChipsInfoKey]: number } = {
+            info: 0,
+            warning: 0,
+            error: 0,
+            none: 0,
+        };
+        const items: React.ReactNode[] = [];
+        accordionData.items.forEach((item) => {
```

Comment:
Related to the comment about the conditional rendering here: https://github.com/mattermost/mattermost-webapp/pull/9693/files/8fdebf5a438c4591fc9c554573e0bac67835c759#r791794668

I would suggest we filter out any items that have `item.status === undefined` before going into that loop.

```suggestion
        accordionData.items.filter((item) => item.status !== undefined).forEach((item) => {
```

Reply:
ok